Deployment/README.md

```diff
@@ -0,0 +1,13 @@
+# Ansible-Kubernetes-Mysql
+Ansible Playbooks use to deploy kubernetes infrastructure with mysql containers. This has been verified on Centos 7.6 64 bit operating systems.
+
+Preapre Two VMs using centos iso file with atleast 2GB Ram, 2 processors and HDD depending on the needs.
+
+Once VM is ready, login into it with root & given password during setup and update with "yum update -y"
+
+After update complete, make and entry of each hosts in /etc/hosts file for name resolution.
+"vi /etc/hosts"
+
+Clone this repository : https://github.com/shailendra-singh93/Ansible-Kubernetes-Mysql.git
+
```

Comment:
It will be good if you can update README file with whatever you have implemented so far.